ui/src/app/views/concept-homepage.tsx

```diff
@@ -407,8 +421,10 @@ export const ConceptHomepage = withCurrentWorkspace()(
                             nextPage={(page) => this.getNextConceptSet(page)}/>
               <SlidingFabReact submitFunction={() => this.setState({conceptAddModalOpen: true})}
                                iconShape='plus'
+                               tooltip={!this.canEdit}
+                               tooltipContent={<div>Requires Owner or Writer permission</div>}
```

Comment:
is this tooltip still relevant in the case where the user has edit permissions but `this.activeSeletedConceptCount` === 0`?

Reply:
good catch.

Reply:
wait this is actually correct, it will only show a tooltip if the user doesn't have edit permission.  (that's what the `tooltip` prop is for-- whether to show or not).  so it will not show up for a user with edit with 0 concepts selected